src/lib/components/amount/amount.svelte

```diff
@@ -14,11 +14,14 @@
   export let showSymbol = true;
   export let multiplier = BigInt(constants.AMT_PER_SEC_MULTIPLIER);
 
+  export let amountClasses = 'typo-text-mono-bold';
+  export let amountPerSecClasses = 'typo-text-small-mono text-foreground-level-4';
+
   $: amountTokenInfo = $tokens && amount ? tokens.getByAddress(amount.tokenAddress) : undefined;
   $: amountPerSecondTokenInfo =
     $tokens && amountPerSecond ? tokens.getByAddress(amountPerSecond.tokenAddress) : undefined;
 
-  function format(amount: Amount) {
+  export function format(amount: Amount) {
```

Comment:
don't think it needs to be exported?

```suggestion
function format(amount: Amount) {
```